### DISCUSSION TITLE
Feature

### DISCUSSION THREAD

**Discussion by lgagne413** [omshub/website#106] [Ideas]
Just had a thought while checking out the progress on the website.  A lot of reviews begin with "As someone new to CS..." or "As someone with 10 years of experience in such-and-such...".  

It could be cool to have this info built into the review somehow (maybe fill out a short profile of themselves).  Then, visitors to the site could filter reviews to see various perspectives.  Like, how do new OMS students review this compared to ones near graduation?  Or, how do people new to Python rate this course?

**Comment by driscoll42** [comment]
I don't think there's a specific issue for it, but it has been a source of considerable discussion about somehow capturing a users background as part of the post, to allow people to filter on people like them. @awpala has a defined structure for that been decided? I've been out of step since the semester started.

**Reply by awpala** [reply]
"Defined" is relative, but I have been a general proponent of collecting this information on review intake, and then figuring out a smart-ish way to filter for this information after the fact...
